src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/constants/Status.kt

```diff
@@ -16,14 +16,22 @@ enum class Status(
   val nomisStatus: String?,
 ) {
   DRAFT("Draft", null),
-  AWAITING_ANALYSIS("Awaiting analysis", "AWAN"),
-  IN_ANALYSIS("In analysis", "INAN"),
-  INFORMATION_REQUIRED("Information required", "INREQ"),
-  INFORMATION_AMENDED("Information amended", "INAME"),
+  AWAITING_REVIEW("Awaiting review", "AWAN"),
+  ON_HOLD("On hold", "INAN"),
+  NEEDS_UPDATING("Needs updating", "INREQ"),
+  UPDATED("Updated", "INAME"),
   CLOSED("Closed", "CLOSE"),
+
+  // TODO: POST_INCIDENT_UPDATE/INCIDENT_UPDATED will be removed
```

Comment:
fyi `INCIDENT_UPDATED` is gone from `preprod` and `prod` but not `dev` so annoyingly can’t be removed yet

Reply:
I'm going to remove from dev both post incident and updated statuses in dev now